src/app/loginComponents/onboarding/downloadcliclient/downloadcliclient.component.ts

```diff
@@ -34,27 +41,28 @@ export class DownloadCLIClientComponent implements OnInit {
     this.dsServerURI = Dockstore.API_URI;
     this.isCopied2 = false;
     let apiVersion = 'unreachable';
+    this.alertService.start('Fetching metadata');
     this.gA4GHService.metadataGet().subscribe(
       (resultFromApi: Metadata) => {
         apiVersion = resultFromApi.version;
         this.dockstoreVersion = `${apiVersion}`;
-        this.downloadCli = `https://github.com/dockstore/dockstore/releases/download/${apiVersion}/dockstore`;
-        this.metadataService
-          .getRunnerDependencies(apiVersion, '3', 'cwltool', 'json')
+
+        // forkJoin returns an array of values, here we map those values to an object
+        forkJoin([this.metadataService.getCliVersion(), this.metadataService.getRunnerDependencies(apiVersion, '3', 'cwltool', 'json')])
           .pipe(finalize(() => this.generateMarkdown()))
           .subscribe(
-            (json: any) => {
-              if (json) {
-                this.cwltoolVersion = json.cwltool;
-              }
+            ([cliInfo, dependencies]) => {
+              this.downloadCli = cliInfo.cliLatestDockstoreScriptDownloadUrl;
+              this.cwltoolVersion = JSON.parse(JSON.stringify(dependencies)).cwltool;
```

Comment:
We call `alertService.start` near the beginning of `ngOnInit`, so  at the end of every path through the subsequent code, we need to call one of alertService methods to signal that the operation is complete.  You've got the calls in both of the error handlers, but you probably need to add a `simpleSuccess` or `detailedSuccess` call to the handler that processes the successfully-retrieved information.
More details here:
https://github.com/dockstore/dockstore-ui2/blob/66af5d00069ed864ffbc19aacdcac2b5337643e2/src/app/shared/alert/state/alert.service.ts#L25

Reply:
added simple success msg